components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -79,6 +82,9 @@ func main() {
 	logger.RegisterSink(lager.NewWriterSink(os.Stdout, lager.DEBUG))
 	logger.RegisterSink(lager.NewWriterSink(os.Stderr, lager.ERROR))
 
+	logger.Info("Registering healthz endpoint for health probes")
+	go health.NewHandler(fmt.Sprintf("%s:%s", cfg.Host, cfg.StatusPort)).Handle()
```

Comment:
**hit:** IMO "Handle" does not say that it will start server, better for me will be "Server". I will also hide starting the go routine with sth like "ServeAsync". Last thing is to maybe hide the `Sprintf` function inside constructor

`healthz.NewHandler(cfg.Host, cfg.StatusPort).ServeAsync()`

The last problem that I have is that you are ignoring the `http.ListenAndServe`, e.g. we can get error when this port will be already allocated. I'm not quite sure if this should be ignored or not 🤔